src/Hooks.cpp

```diff
@@ -625,9 +625,5 @@ void InstallPreHooks()
 {
 	std::memset(&Player::fakeHealth, 255, sizeof(Player::fakeHealth));
 	std::memset(&Player::fakeArmour, 255, sizeof(Player::fakeArmour));
-
-	for (int i = 0; i < 1000; i++)
-	{
-		Player::fakeQuat[i] = 0;
-	}
+	std::memset(&Player::fakeQuat[0], NULL, sizeof(Player::fakeQuat));
```

Comment:
https://en.cppreference.com/w/cpp/algorithm/fill

Reply:
> You should generally avoid allocating each `glm::quat` into heap.

Sure, JustMichael can fix that. This pull request was made to fix a memory leak in SKY's current approach to store quaternions. I, however, agree with you.

> https://en.cppreference.com/w/cpp/algorithm/fill

memset is faster (or equally as efficient) than `std::fill`. They use memset 2 times above too, so I think bringing `std::fill` there just to, well, fill an array would be worthless when we're managing POD. If I were to use `std::fill`, I'd also use `std::array` and `std::array::fill`.